File: MobileAppReactNativeMobX/__tests__/utils.test.ts
import 'react-native';
import { expect, test } from '@jest/globals';
import { round } from '../src/business/utils/utils';

test('round', () => {
  expect(round(1.2345)).toBe(1.23);
  expect(round(1.236)).toBe(1.24);
  expect(round(1.234)).toBe(1.23);
  expect(round(1.23)).toBe(1.23);
  expect(round(1)).toBe(1);
  expect(round(-1.2345)).toBe(-1.23);
});

File: MobileAppFlutterCelest/test/utils_test.dart
import 'package:celest_backend/models.dart';
import 'package:flutter_test/flutter_test.dart';

void main() {
  test('round', () {
    expect(round(1.2345), 1.23);
    expect(round(1.23), 1.23);
    expect(round(1.239), 1.24);
    expect(round(0), 0);
    expect(round(-1.2345), -1.23);
  });
}
